packages/preview/muw-touying/0.1.0/README.md

```diff
@@ -0,0 +1,61 @@
+# Typst/Touying Template for MedUni Wien
+
+This is a template built on [`Touying`](https://github.com/touying-typ/touying) to create presentation slides that capture the design of the official PowerPoint templates following Medizinischen Universität Wien [Corporate Design](https://www.meduniwien.ac.at/web/en/studierende/service-center/meduni-wien-vorlagen/).
+
+## Disclaimer
+
+*This theme is __unofficial__. It is __NOT__ affiliated with Medizinischen Universität Wien (Medical University of Vienna). The logo is the intellectual property of the Medical University of Vienna and is subject to copyright. Users are advised to check the official Styleguide before using this template.*
+
+## Example
+
+See [examples](examples) for more details.
+
+## Usage
+
+Use the following code as a starting point to create your own slides.
+
+```typst
+#import "@preview/touying:0.6.1": *
+#import "lib.typ": *
+
+#set text(lang: "en")
+#show: muw-slides.with(
+  config-common(slide-level: 2),
+  config-info(
+    title: [Title slide with a blue/white background],
+    author: [Univ. Prof. Dr. Peter Strasser],
+    institution: [Universitätsklinik für XY],
+    organization: [Medizinische Universität Wien],
+  ),
+  page-numbering-start: 2,
+)
+
+
+// Your slide starts here
+
+#title-slide-dunkelblau()
+
+= Section Header
+Version – white background
+
+== Title of the slide
+
+=== Subtitle
+
+Hello, MedUni Wien!
+
+```
+
+## License and copyright
+
+- This template builds upon `Touying` and is licensed under [MIT license](LICENSE).
+- The MedUni Wien logo is the intellectual property of the Medical University of Vienna and is subject to copyright.
+- Some beautiful pictures used in the example template are copyright of Nihon Falcom Corporation and are used for demonstration purpose only.
```

Comment:
Regarding this last point, I'm not sure you are allowed to distribute these images even if you credit them. Unless I'm mistaken, can you please replace them with other images?

Reply:
Thanks for the suggestions. I’ve replaced these images with other images that are licensed to allow distribution. The `README.md` file is adjusted accordingly.